### ISSUE TITLE
[Updatecli/kubernetes-management] Improve the Maven manifest.

### ISSUE THREAD

**Issue by smerle33** [jenkins-infra/helpdesk#4283]
while working on https://github.com/jenkins-infra/kubernetes-management/pull/5666 we saw that some improvements can be done on the Maven manifest. Some of these improvements are foundational for new updatecli manifest and worth a top level issue.

- [x] the log for the updatecli diff is way too wordy, it can be shorten by using `matchpattern` as per the second source example.
- [x] move the manifest to the top updatecli.d folder as it's not related to a chart
- [x] @dduportal 's  comment about a missing feature is a mistake and can be removed.

**Comment by jayfranco999** [comment]
Update:

https://github.com/jenkins-infra/kubernetes-management/pull/5717

The log for the updatecli diff has been cleaned up by using a better match pattern.

The manifest has been move to the top updatecli.d folder.
